src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1118,6 +1122,64 @@ int NetworkOPsImp::findTransactionsByDestination (
 // Account functions
 //
 
+bool NetworkOPsImp::getAccountObjects (Ledger::pointer lpLedger,
+    Account const& accountID, uint256 const& startAfter,
+    std::uint64_t const hint, unsigned int limit,
+    std::function <bool (SLE::ref)> func) const
+{
+    auto const rootIndex (getOwnerDirIndex (accountID));
+    auto currentIndex (rootIndex);
+    bool found (true);
+
+    if (startAfter.isNonZero ())
+    {
+        found = false;
+
+        auto const hintIndex (getDirNodeIndex (rootIndex, hint));
+        SLE::pointer hintDir (lpLedger->getSLEi (hintIndex));
+        if (hintDir != nullptr)
+        {
+            for (auto const& node : hintDir->getFieldV256 (sfIndexes))
+            {
+                if (node == startAfter)
+                {
+                    // We found the hint, we can start here
+                    currentIndex = hintIndex;
+                    break;
+                }
+            }
+        }
+    }
+
+    for (;;)
+    {
+        auto const sleNode (lpLedger->getDirNode (currentIndex));
+
+        if (sleNode == nullptr)
+            return found;
```

Comment:
So if `startAfter == zero` and we don't have a node at the current index, we return `true`.  Is that right/what does that mean?